ui/src/ElasticSearch.elm

```diff
@@ -0,0 +1,114 @@
+module ElasticSearch exposing (..)
+
+import Http
+import Json.Decode as Decode
+import Json.Encode as Encode
+import Types exposing (..)
+
+
+filterToJsonProperty : String -> Maybe String
+filterToJsonProperty filter =
+    case filter of
+        "product" ->
+            Just "source.product"
+
+        "channel" ->
+            Just "target.channel"
+
+        "locale" ->
+            Just "target.locale"
+
+        "version" ->
+            Just "target.version"
+
+        "platform" ->
```

Comment:
Yeah this was hacked in a hurry. I'd love filters to be an array of json field names indeed, but maybe I can address that in a next PR.